src/components/list-item/list-item.tsx

```diff
@@ -28,9 +46,10 @@ export class ListItem implements ConditionalSlotComponent, InteractiveComponent
   // --------------------------------------------------------------------------
 
   /**
-   * When true, prevents the content of the component from user interaction.
+   *
+   * @internal
    */
-  @Prop({ reflect: true }) nonInteractive = false;
```

Comment:
Similar to `headingLevel`, we could deprecate this, and not use it, to avoid breaking changes on developers setting it to the default for some reason.